config/routes.rb

```diff
@@ -1,7 +1,17 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  root 'tasks#index'
-  get 'tasks', to: 'tasks#index'
-  get 'tasks/overview'
+  root 'application#index'
+
+  resources :task_lists do
+    member do
+      get :delete
```

Comment:
This shouldn't be needed:
1) It should be `delete` request, not `get`
2) Action should be `destroy`

If you will have `resources :task_lists` it will generate automatically all the necessary routes for you,
see https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions